vim/vscode-code-actions.vim

```diff
@@ -65,18 +65,22 @@ nnoremap <expr> = <SID>vscodeFormat()
 nnoremap <expr> == <SID>vscodeFormat() . '_'
 
 " gf/gF . Map to go to definition for now
+nnoremap <silent> K :<C-u>call VSCodeNotify('editor.action.showHover')<CR>
 nnoremap <silent> gh :<C-u>call VSCodeNotify('editor.action.showHover')<CR>
 nnoremap <silent> gf :<C-u>call <SID>vscodeGoToDefinition("Declaration")<CR>
 nnoremap <silent> gd :<C-u>call <SID>vscodeGoToDefinition("Definition")<CR>
 nnoremap <silent> <C-]> :<C-u>call <SID>vscodeGoToDefinition("Definition")<CR>
+nnoremap <silent> gO :<C-u>call VSCodeNotify('workbench.action.gotoSymbol')<CR>
```

Comment:
can you create a jumppoint also here? move to custom function and do `normal! m'` before

Reply:
Does this do what ctrl-p followed by `@` does? That is a much more usable "outline" of the current symbols than vscode's "outline" pane.

Reply:
>Does this do what ctrl-p followed by @ does? 

Yes, also `cmd/ctrl-shift+o` . Display quickpick with symbols

Reply:
Current shortcut works like ctrl+p followed by # (displays current document symbols). You mean add a shortcut for `ctrl+shift+p`?

Reply:
> You mean add a shortcut for `ctrl+shift+p`?

No.

> Yes, also `cmd/ctrl+shift+o` . Displays quickpick with symbols

Great

Reply:
Sorry, that was a typo :) I meant to add a shortcut for `ctrl+shift+o`.

Reply:
>I meant to add a shortcut for ctrl+shift+o

Won't hurt i guess, since we're already "enhancing" few vscode default keybindings. Enhancing point is jumppoint creation here

Reply:
Done. How this can be done for VSCode shortcuts?

Reply:
```json
            {
                "command": "vscode-neovim.send",
                "key": "ctrl+shift+o",
                "when": "editorTextFocus && neovim.mode != insert",
                "args": "<C-O>"
            },
            {
                "command": "vscode-neovim.send",
                "key": "cmd+shift+o",
                "when": "editorTextFocus && neovim.mode != insert",
                "args": "<C-O>"
            }
```

Reply:
Why is adding that vscode shortcut needed? "ctrl+shift+o" already works fine without needing an explicit binding. And did you mean `"args": "<C-S-O>"`?

Reply:
Will Neovim distinguish <C-O> from <C-o>? In Neovim Qt and Konsole `<C-O> do not works.

Reply:
> Why is adding that vscode shortcut needed? "ctrl+shift+o" already works fine without needing an explicit binding. 

Currently we only trying to create jumppoint on `ctrl+shift+o`. `gO` is like `ctrl+p` with `#` (there is no default shortcut for it and gO is basically corresponds to the document symbols).

Reply:
>Why is adding that vscode shortcut needed? "ctrl+shift+o" already works fine without needing an explicit binding. 

It won't create the jump point. So `gO` will while `ctrl+shift+o` won't, a little inconsistency here.

>And did you mean "args": "<C-S-O>"?

👍